Bluejay/Bluejay/Bluejay.swift

```diff
@@ -663,11 +677,18 @@ extension Bluejay: CBCentralManagerDelegate {
         UIApplication.shared.endBackgroundTask(backgroundTask)
     }
     
+    /**
+     Figure out whether this is any kinds of disconnection when Bluejay is still connecting, an expected disconnection from a connected state, or an unexpected disconnection from a connected state. An unexpected disconnection from a connected state, such as walking out of range or restarting the Bluetooth device, is the only case where `shouldAutoReconnect` should be set to true.
```

Comment:
Yes, I am not sure why these delegate functions are public. Might be something left over from the past. I'll file a new issue and open a new PR for making CBCentral related delegation internal.